src/plugins/dashboard/public/saved_dashboards/saved_dashboard.ts

```diff
@@ -98,6 +100,7 @@ export function createSavedDashboardClass(
           panelsJSON: { type: 'text' },
         },
       },
+      ...userContentCommonMappings,
```

Comment:
I am not sure why we have duplicate mappings definition with the server. This would probably need to be refactored to have a single mappings declaration